ORK1Kit/ORK1Kit/Common/CEVRK1Theme.h

```diff
@@ -12,34 +12,71 @@
 
 extern NSNotificationName _Nonnull const CEVORK1StepViewControllerViewWillAppearNotification;
```

Comment:
We probably can remove this now since we are no longer using it's hackery :)

Reply:
Yup removed.